sites/friday/layouts/shortcodes/guides/deploy/configure/project-init.md

```diff
@@ -0,0 +1,61 @@
+## Add {{ .Site.Params.vendor.name }} configuration files
+
+Before attempting the next commands, make sure you have the {{ .Site.Params.vendor.name }} CLI installed, working, and authenticated with your {{ .Site.Params.vendor.name }} account.
+
+1. In your local repository, create a new branch
+
+    ```bash
+    git checkout -b {{ .Site.Params.vendor.cli }}ify
+    ```
+
+1. Run the command:
+
+    ```bash
+    {{ .Site.Params.vendor.cli }} project:init
+    ```
+
+    The {{ .Site.Params.vendor.name }} CLI will now ask you a series of questions to determine your project’s requirements:
+
+    ```bash
+    $ {{ .Site.Params.vendor.cli }} project:init
+
+    Welcome to Upsun!
+    
+    Let's get started with a few questions.
+
+    We need to know a bit more about your project. This will only take a minute!
+    ```
+
+<!-- Framework-specific -->
+
+{{ .Inner }}
+
+<!-- End of framework specific -->
+
+1. **Commit & push**. With your configuration in place, commit the changes:
+
+    ```bash
+    git add . && git commit -m "Add {{ .Site.Params.vendor.name }} configuration."
+    ```
+
+    Then push to your repository:
+
+    ```bash
+    {{ .Site.Params.vendor.cli }} push
+    ```
+
+    If the CLI asks you follow-up questions at this point, be sure to choose `{{ .Site.Params.vendor.cli }}ify` as the target branch, but choose **No** if it asks you to activate the environment. 
+    You will do this in a later step.
+
```

Comment:
Assuming the user hangs around in their terminal until the upsun push command finishes, should we warn them that a failure is _expected_?